test/graphdigraph.jl

```diff
@@ -22,10 +22,10 @@ add_edge!(h, 2, 5)
 add_edge!(h, 3, 5)
 
 
-@test_throws ErrorException add_edge!(g, 1, 2)
-@test_throws ErrorException add_edge!(h, 1, 2)
-@test_throws ErrorException add_edge!(g, 1, 1)
-@test_throws ErrorException add_edge!(h, 1, 1)
+# @test_throws ErrorException add_edge!(g, 1, 2)
+# @test_throws ErrorException add_edge!(h, 1, 2)
```

Comment:
Lines 25 and 26 should still remain, no? They're not self-loops - they're duplicate edges.